src/uv.cc

```diff
@@ -75,6 +76,7 @@ void ErrName(const FunctionCallbackInfo<Value>& args) {
   CHECK_LT(err, 0);
   const char* name = uv_err_name(err);
   args.GetReturnValue().Set(OneByteString(env->isolate(), name));
+  freeUVErrorNameMemoryIfNeed(name, err);
```

Comment:
We should move to [`uv_err_name_r`](http://docs.libuv.org/en/v1.x/errors.html?highlight=uv_err_name#c.uv_err_name_r) instead of introducing this workaround.

Reply:
Done !